app/models/user.rb

```diff
@@ -9,4 +9,27 @@ class User < Base
   def self.member(id)
     new(id: id)
   end
+
+  def self.generate_and_send_magic_link(email)
```

Comment:
This doesn't seem like a `User` concern. Could this be a service?

Reply:
Ya, I was being lazy / expedient. :)